index.js

```diff
@@ -50,7 +50,8 @@ var rules = {
   'jsx-no-target-blank': require('./lib/rules/jsx-no-target-blank'),
   'jsx-filename-extension': require('./lib/rules/jsx-filename-extension'),
   'require-optimization': require('./lib/rules/require-optimization'),
-  'no-find-dom-node': require('./lib/rules/no-find-dom-node')
+  'no-find-dom-node': require('./lib/rules/no-find-dom-node'),
+  'no-children-prop': require('./lib/rules/no-children-prop')
```

Comment:
I agree, but that change could be considered breaking whereas this change is minor. If this is going to make it into v6 then that's fine, though.